bin/phase-helpers.sh

```diff
@@ -853,6 +853,23 @@ __eapi6_src_install() {
 	einstalldocs
 }
 
+__eapi8_src_prepare() {
+	local f
+	if ___is_indexed_array_var PATCHES ; then
+		for f in "${PATCHES[@]}"; do
+			[[ ${f} == -* ]] && die "Options disallowed in PATCHES array"
+		done
```

Comment:
That's not as specified. `PATCHES` should just be passed to eapply without further filtering, i.e., all elements in `PATCHES` are supposed to be filenames, even if they start with `-`. (That's also how POSIX tools behave, `--` suppresses special treatment of `-` for all following arguments.)

(I think we don't want diverging behaviour between the default and explicit `eapply -- ${PATCHES}` in an ebuild.